### DISCUSSION TITLE
Way to remove Overclocks?

### DISCUSSION THREAD

**Discussion by DeviousDaniel** [AnthonyMichaelTDM/DRG-Save-Editor#27] [Q&A] [answered]
So I'll preface this by saying I noticed this error way too late and don't have access to my very old backups from before I made this error. Probably should have kept all my backups, even from months ago, but alas.

Back when this was still an online tool, I tried to use it to give myself a couple Overclocks. Now, after playing legit for several months, I've stopped receiving any Weapon Overclocks from events or assignments. Basically, the game thinks I have all the overclocks, but I'm still missing a few. Namely, I'm missing the ones that I tried adding from the online save editor all that time ago.

So, I found and downloaded this version of the save editor and, lo and behold, it lists all those missing overclocks as "forged". So somehow they are "in my inventory" even though they aren't visible in-game. Maybe there is some flag that got checked off behind the scenes to tell the game that I have them, but I can't actually access them.

So is there any solution for this? Is there a way to change the status to "Unacquired" or something so I can just find them the normal way? Or am I poetically forever boned and will never get to use those Overclocks I tried to cheat in?

**Comment by AnthonyMichaelTDM** [comment, marked as answer]
First and foremost, this was never an online tool, you must be thinking of something else?

Now, to address your question, this issue you're experiencing is precisely why this tool adds OC's as "unforged" and also why it automatically creates backups before saving any changes.

As for resolving your issue, this tool does not allow the removal of "forged" OCs, as that would cause cascading issues if, say, you had the OC equipped to something. In your case, you'll likely have to edit the save file manually, or modify your installation of the save editor to allow removal of forged OC's (it'll be a little involved either way).

To summarize, you aren't exactly boned, but will need to do some work to fix it.

**Reply by DeviousDaniel** [reply]
Thanks for the advice! Yeah I must be thinking of a different save editor. One that was an online tool temporarily until the DRG devs asked them to take the site down.

Is there a tutorial I can find out there to help me manually edit the save file? I'm no luddite, but I'm definitely not a software dev.

**Reply by AnthonyMichaelTDM** [reply]
You can look at the get_overclocks function to see how it finds the number of forged oc's and what the OCs are, as for tutorials, idk, you could try YouTube?

I think the hex editor I used was HxD

**Reply by AnthonyMichaelTDM** [reply]
Be sure to make backups this time lol, you don't want to actually nuke your save on accident

**Reply by AnthonyMichaelTDM** [reply]
Also de-equip all weapon overclocks to be safe, you can cross reference the guids.json file to figure out what is and isn't a cosmetic OC because you don't want to delete those

**Reply by AnthonyMichaelTDM** [reply]
Another thing you can do is diff your save file with someone else's to see how everything is laid out a little clearer

**Reply by wolfnacht-labs** [reply]
Excuse me for replying to this thread a year later, feel free to delete my comment or whatever if that's not in the site's culture, but I tried using HxD and deleting the string responsible for Magnetic Pellet Alingment (which is 07E9279956BF9C418EEA313E2DC59F8D according to guids.json file) and the whole save gets bricked, I'm reset to 0 progress (level 1 and all that), as if this was a fresh save. Am I doing something wrong? HxD warns me about file size change after making that edit.

**Reply by AnthonyMichaelTDM** [reply]
yeah, you'd have to replace it with another oc, or theres another number that indicates the number of oc's and that needs to be updated if you change the number of ocs
